### ISSUE TITLE
Acc is always 1.00 during training my own dataset

### ISSUE THREAD

**Issue by Davy4fg** [MarvinTeichmann/KittiSeg#155]
Hi Marvin,

Thanks for sharing, the training on your dateset and evaluate is working fine.

But after using my own dataset for pig segmentation, the acc is always 1.00 during training. 
The complete image will be regard as pig. No background.
Do you have any idea what could be the problem?

2018-01-26 21:11:36,775 INFO Step 0/2000: loss = 0.46; lr = 1.00e-03; 0.082 sec (per Batch); 12.2 imgs/sec
2018-01-26 21:11:37,892 INFO       (raw)  Acc. : 1.00, xentropy: 0.00, weight_loss: 0.38
2018-01-26 21:11:37,892 INFO    (smooth)  Acc. : 1.00, xentropy: 0.00, weight_loss: 0.38

Thanks for answering in advance.

**Comment by Davy4fg** [comment]
Issue solved. The training data doesn't meet the need, some seemly correct color is not really correct. Like R is 254 instead of 255.
